### DISCUSSION TITLE
stepper_move with Negative number has problem

### DISCUSSION THREAD

**Discussion by bloodsuckant** [MrYsLab/telemetrix#18] [Q&A] [answered]
Hi, thanks for this beautiful project. I use Telemetrix to control a stepper motor. The stepper_move_to is nice, it working correctly. However, the  stepper_move with negative number is not simply counter clockwise from the current position, it will go counter clockwise with extra 65536 steps, i.e. -(65536 + distance_to_go) steps. It is nice if u can fix this. For now, people can use stepper_move_to (absolute) to control the motor nicely.

**Comment by bloodsuckant** [comment, marked as answer]
Sorry, I find the answer, Just use global variable to pass the value out.

**Reply by MrYsLab** [reply]
@bloodsuckant 
Glad you found the answer. Since this wasn't a bug, I have removed the issue. If you think there is still an issue, please let me know. Thanks again.

**Reply by bloodsuckant** [reply]
@MrYsLab 
stepper_move with Negative number has problem is still a problem. The reply is answered another discussion which is also create by me. It goes to the wrong post (and I cannot find the post now, maybe I didn't create it successfully) .  Please tag this disscusion as an issue. Thanks a lot!

**Comment by MrYsLab** [comment]
@bloodsuckant I have a fix available for Arduino boards. Before publishing, I will also need to test the ESP-8266 for both asyncio and non-asyncio telemetrix clients before publishing the changes.

I have entered issue #22 to track the issue.

However, I am currently completing my work on Telemetrix for the ESP32, which should take about one to two weeks. Once this is done, I will publish the fixes for the Arduino.

**Reply by MrYsLab** [reply]
Quick update. I am still working on ESP32 BLE. I will probably need another week to two before I can fix this issue.

**Reply by MrYsLab** [reply]
Fixed with version 1.9 and published to pypi.
